content/docs/components/heading/theming.mdx

```diff
@@ -2,3 +2,160 @@
+
+The properties that affect the theming of the `Heading` component are:
+
+- `variant`: The visual variant of the divider.
+- `colorScheme`: The color scheme of the divider.
+- `size`: The size of the divider. Defaults to `xl`.
+
+## Theming utilities
+
+- `defineStyle`: a function used to create style objects.
+- `defineStyleConfig`: a function used to define the style configuration for a single part component.
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+```
+
+## Customizing the default theme
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const custom = defineStyle({
+    color: "yellow.500",
+    border: "0px",
+    // let's also provide dark mode alternatives
+    _dark: {
+        color: 'yellow.300',
+    }
+})
+```
+After customizing the divider theme, we can import it in our theme file and add it in the components property:
+
+```js
+import { extendTheme } from '@chakra-ui/react'
+import { headingTheme } from './components/divider.ts'
+
+export const theme = extendTheme({
+    components: { Heading: headingTheme },
+})
+```
+
+> This is a crucial step to make sure that any changes that we make to the divider theme are applied.
+
+Every time you're adding anything new to the theme, you'd need to run the cli command to get proper autocomplete in your IDE. Here's how you can do that:
+
+```bash
+# run it once:
+npx chakra-cli tokens path/to/theme.ts
+# or run it in watch mode:
+npx chakra-cli tokens path/to/theme.ts --watch
+```
+
+## Adding a custom variant
+
+Let's assume we want to include a custom branded variant. Here's how we can do that:
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const thicc = defineStyle({
+    color: "orange.500",
+    border: "0px",
+    bg: "blue.200",
+    p: "3",
+    borderRadius: "10",
+    transition: 'transform 0.15s ease-in-out, background 0.15s ease-in-out',
+    // let's also provide dark mode alternatives
+    _dark: {
+        color: 'orange.400',
+        bg: "blue.200"
+    },
+    _hover: {
+        bg: "blue.100",
+        color: "purple.300",
+        transform: "scale(1, 1)",
+        _dark: {
+            bg: "blue.100",
+            color: "purple.300",
+            transform: "scale(3, 3)",
+        }
+    }
+})
+
+// Now we can use the new `thicc` variant
+<Divider variant="thicc">...</Divider>
+```
+
+## Using a custom color scheme
+
+Let's assume we want to use our own custom color scale based on our brand. We'd need to define the color scale first in the main theme file:
+
+```js
+import { extendTheme } from '@chakra-ui/react'
+export const theme = extendTheme({
+    colors: {
+        brand: {
+            50: '#f7fafc',
+            ...
+            500: '#718096',
+            ...
+            900: '#171923',
+        }
+    }
+})
+```
+
+Then, we can use the custom color scale as the color scheme for the divider:
+
+```jsx
+<Divider colorScheme='brand'>...</Divider>
+```
```

Comment:
No variant utilizes the `colorScheme`, so I'd either remove this, or create a custom variant that utilizes the `colorScheme` and provide examples for it. Also, you've got `Divider` here as well.